lib/avromatic/schema_registry_patch.rb

```diff
@@ -7,24 +7,26 @@ module CacheableSchemaRegistration
     def register(subject, schema)
       return super unless Avromatic.use_cacheable_schema_registration
 
+      begin
+        lookup_subject_schema(subject, schema)
+      rescue
```

Comment:
Fair enough, I verified that our server returns a 404 for an unsupported endpoint, and I tested an instance of the Confluent Schema Registry and it also returns a 404 for this request.